modules/core/dictionaries/no_state.translation.json

```diff
@@ -83,6 +83,9 @@
 		"cs": "N\u00e1vrhy pro vy\u0159e\u0161en\u00ed tohoto probl\u00e9mu:",
 		"eu": "Arazo hau konpontzeko iradokizunak:"
 	},
+	"suggestion_badlink": {
```

Comment:
Translations are handled through the [translation portal](https://translation.rnd.feide.no/), not directly in the code, so you don't need to provide them here. We just have to update the strings in the portal so that you can translate them there, and afterwards we'll be able to generate updated translations to be shipped with SimpleSAMLphp.